arrow-optics/src/test/kotlin/arrow/optics/instances/SetInstanceTest.kt

```diff
@@ -36,7 +37,7 @@ class SetInstanceTest : UnitSpec() {
 
     testLaws(
       LensLaws.laws(
-        lensGen = Gen.string().map { setAt<String>().at(it) },
+        lensGen = Gen.string().map { At.set<String>().at(it) },
```

Comment:
🤔  

Updated to use the new method in tests. However, there is another test in this file that uses `SetK.at<String>()` that is deprecated. Should I leave it as it is until everything gets removed? Should it be replaced with something else?

Reply:
There is an alternative for Set that has been updated. Nothing else is required for SetK, when it is removed the tests will also be removed.